core-bundle/tests/EventListener/BackendCacheResponseListenerTest.php

```diff
@@ -151,7 +151,7 @@ public function testSetsVaryHeader(): void
 
         (new BackendCacheResponseListener($this->createScopeMatcher(true)))($event);
```

Comment:
Should we add a previous `vary` header just to actually test it's not replaced but added?